promisasu/DMTBPortal
# Task#147

**Taiga User Story** - Story #140

**Taiga Task(s)** - Task#147

**What did you change?** - Created target to stop portal.

**How can we test?** 
- start portal using ant startportal
- check functionality : run ant stopportal
